src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -877,34 +877,36 @@ public void readFromNBT(final CompoundNBT data) {
         this.getFacadeContainer().readFromNBT(data);
     }
 
-    public List<ItemStack> getDrops(final List<ItemStack> drops) {
+    /**
+     * Append the contents of the parts to the drop list, e.g. crafting patterns but not the parts or facades
+     * themselves.
+     */
+    public void appendPartContentDrops(final List<ItemStack> drops) {
```

Comment:
This probably needs a better name.  And overall I would prefer to handle it within the part itself by passing the loot context or similar around and have the parts deal with it and not have a bunch of different code paths, but this requires an API break.

Reply:
Should we have a list of possible API breaks for 1.17 somewhere so we don't forget about them?

Reply:
What about `appendPartInventories`?

Reply:
Marking as resolved.